src/main.js

```diff
@@ -1,28 +1,33 @@
 import Vue from 'vue'
 import Buefy from 'buefy'
+import 'buefy/dist/buefy.css'
 import i18n from './i18n'
 import VueAnalytics from 'vue-analytics'
 import App from './App.vue'
 import store from './store'
 
 import { library } from '@fortawesome/fontawesome-svg-core'
-// internal icons
-import {
-    faCheck, faCheckCircle, faInfoCircle, faExclamationTriangle, faExclamationCircle,
-    faArrowUp, faAngleRight, faAngleLeft, faAngleDown, faUserCircle, faQuestionCircle,
-    faEye, faEyeSlash, faCaretDown, faCaretUp, faUpload, faCopy, faCircle
-} from '@fortawesome/free-solid-svg-icons'
+import { faCopy } from '@fortawesome/free-solid-svg-icons/faCopy'
```

Comment:
I would prefer going back to the 
```
import {
	dasfasd, asdfasfd, asdfsafda, safdsafd, asdfafsad
} from '@fortawesome/fre-fadsdf-asdfasdf-dsfafs'
```
syntax. No need to re-build the site after this change :)